addons/website/views/snippets/snippets.xml

```diff
@@ -1218,10 +1218,8 @@
                           t-attf-class="o_we_user_value_widget o_we_gray_preview bg-#{grayCode}"/>
                 </t>
             </we-row>
-            <we-range string="⌙ Hue" data-customize-gray="" data-min="0" data-max="360"
-                data-param="gray-hue" class="o_we_slider_tint"/>
-            <we-range string="⌙ Saturation" data-customize-gray="" data-min="-20" data-max="100"
-                data-param="gray-extra-saturation"/>
+            <we-range string="⌙ Hue" class="o_we_slider_tint" data-customize-gray="" data-param="gray-hue" data-min="0" data-max="359.9" data-step="0.1"/>
+            <we-range string="⌙ Saturation" data-customize-gray="" data-param="gray-extra-saturation" data-step="0.1"/>
```

Comment:
@oussamamessaoudi Found another potential thing that led to rounding errors (and so UI "jumps" when choosing a value) -> the precision of the slider itself. I added a step of 0.1 instead of the default 1. I also set the max hue value to 359.9 instead of 360 (to avoid the slider to systematically jump back to 0 once 360 was chosen... stilll can happen depending on rounding errors due to the chosen saturation though).